src/settings_pack.cpp

```diff
@@ -266,7 +266,7 @@ namespace libtorrent {
 		SET(auto_scrape_min_interval, 300, nullptr),
 		SET(max_peerlist_size, 3000, nullptr),
 		SET(max_paused_peerlist_size, 1000, nullptr),
-		SET(min_announce_interval, 5 * 60, nullptr),
+		SET(min_announce_interval, 2 * 60, nullptr),
```

Comment:
what's the rationale for this change?

Reply:
The Websocket tracker requires a shorter announce period, because WebRTC offer batches are piggybacked on announces. The default value for the interval sent in the tracker response in 2 min. If that's not OK to reduce the value, what about a different value for Websocket trackers?

Reply:
yeah, I think a separate setting would be better. since it's controlling a separate kind of thing

Reply:
I added a different `min_websocket_announce_interval` setting.